### PR TITLE
Serve /.well-known/acme-challenge/ for http-01 challenge.

### PR DESCRIPTION
**Pull request by ShaneMcC**
Issue csmith/docker-automatic-nginx-letsencrypt#3

### CODE REVIEW

**Comment by ShaneMcC** [comment]
Related: csmith/docker-automatic-nginx-letsencrypt#10

File: nginx.tpl [hunk 1]
```diff
@@ -10,6 +10,19 @@ server {
 
     include /etc/nginx/conf.d/{{ service.vhosts[0] }}/*.conf;
 
+    # From https://community.letsencrypt.org/t/how-to-nginx-configuration-to-enable-acme-challenge-support-on-all-http-virtual-hosts/5622
+    location ^~ /.well-known/acme-challenge/ {
+        default_type "text/plain";
+        alias /letsencrypt/well-known/;
```
Comment (csmith) [inline nginx.tpl]: I don't think this should be hardcoded, should be passed in as an argument (sorry!)

**Review by csmith — CHANGES_REQUESTED** [review]

**Comment by ShaneMcC** [comment]
Probably needs rebasing on top of #3 and #4

**Comment by ShaneMcC** [comment]
Rebased on #4

**Pull request merged**